### ISSUE TITLE
Unable to import Query/Mutation from nested schemas

### ISSUE THREAD

**Issue by LawJolla** [graphcool/graphql-import#134]
E.g.
```
# schema.graphql
# import Mutation.createUser from "....../generated/prisma.graphql"
....
# import * from "./post/post.graphql"
...
```

```
# post.graphql
# import Mutation.createPost from "....../generated/prisma.graphql"
...
```

will not import Mutation.createPost or throw an error.  Please let me know if an example repo would help.  Thank you!

**Comment by SpaceK33z** [comment]
Hey, this is a duplicate of #115. I made a PR to fix this issue, see #140.

**Comment by schickling** [comment]
This should be fixed by now thanks to @SpaceK33z! 🎉
